templates/web/common/site_layout.tt.html

```diff
@@ -490,6 +490,14 @@ <h3 class="title-5 text-medium">[% lang('footer_discover_the_project') %]</h3>
 [% END %]
 
 </body>
+<script src="[% static_subdomain %]/js/dist/JsBarcode.all.min.js"></script>
+<script>
+	//init barcode SVGs if exist on the page
```

Comment:
I'm not sure what the best place for this snippet is, but in general, we tried to remove most direct JS from the templates in the last few months. Maybe [display-product.js](https://github.com/openfoodfacts/openfoodfacts-server/blob/21681e9e25fcae0eba17ddd82da466149e522834/html/js/display-product.js)? cc @alexgarel @stephanegigandet

Reply:
display-product.js would be the right place I think, we don't need to load this script on every page, just product pages.

In fact we probably want to display the barcode only on desktop (not mobile), as on mobile it takes a lot of space for little added value. On desktop it's very useful so that it can be scanned with a mobile app (like ours, or others).

I'm not sure if we could load the script conditionnaly if the screen is big enough though.